### ISSUE TITLE
Are you sure this is needed?

### ISSUE THREAD

**Issue by andrerpena** [redux-autoform/redux-autoform.github.io#1]
I mean, people normally use the `gh-pages` branch of the main repository to host the page and demo... Is there any specific need for a separate repository?

**Comment by JonatanSalas** [comment]
A homepage for redux-autoform would be cool, this could be the main page for the organization, and then from this page redirect to all demos in the ui repos. 

Instead of presenting each demo, we could present the project itself first. 

Do you understand my point? If you don't want this we could delete the repo :) There's no trouble!

**Comment by andrerpena** [comment]
Yes, one advantage is that we could create a single demo infrastructure instead of having that replicated among different UI projects. 

It's ok for me. We can do it what way :)
